frontend/app/forms/motion-to-reduce-conviction/motion-to-reduce-conviction.web.component.js

```diff
@@ -1,5 +1,83 @@
 import React from "react";
+import FormThatPrints from "../inputs/form-that-prints.component";
+import Section from "../inputs/section.component.js";
+import TextInput from "../inputs/text-input.component";
+import Select from "../inputs/select.component.js";
+import Radio from "../inputs/radio.component.js";
+import GroupSelect from "../inputs/group-select.component.js";
+import {
+  petitionerRepresentativeOptions,
+  courtTypeOptions,
+  JusticeCourtList,
+  DistrictCourtList
+} from "../form-common-options/form-common-options";
 
-export default function MotionToReduceConviction_Web() {
-  return <div>Web component</div>;
+export default function MotionToReduceConviction_Web({ data }) {
+  return (
+    <FormThatPrints>
+      <Section name="1. Personal Information">
+        <TextInput dataKey="person.firstName" label="First Name" />
```

Comment:
I think we have a firstName in the `en.js`

`<TextInput dataKey="person.firstName" label={__("first name")} />`

Same for line 21.

Reply:
great catch! thanks!